doc/src/build/move/index.md

```diff
@@ -37,13 +37,12 @@ to deepen your understanding of the Move language but not strict prerequisites
 to following the Sui tutorial, which we strived to make self-contained.
 Further, Sui does differ in some ways from Move, which we explore here.
 
-In Sui, Move is used to define, create and manage programmable Sui
-[objects](../objects.md) representing user-level assets.  Sui
-imposes additional restrictions on the code that can be written in
-Move, effectively using a subset of Move (a.k.a. *Sui Move*), which
+In Sui, Move is used to define, create, and manage programmable [Sui objects](../objects.md) representing user-level assets.  Sui
+imposes additional restrictions on code written in
+Move, creating a dialect of Move (a.k.a. *Sui Move*) that
```

Comment:
Trying to add a more additive flair to this wording, thoughts? Maybe something like:
```suggestion
In Sui, Move is used to define, create, and manage programmable [Sui objects](../objects.md) representing user-level assets.  Sui's object system is implemented by adding new functionality to Move while also imposing additional restrictions, creating a dialect of Move (a.k.a. *Sui Move*) that
```

Reply:
Just saw @sblackshear's comment so maybe it's not critical to worry about the wording at this point in time

Reply:
I like this. No harm in updating and merging later updates with it. Thanks!